### PR TITLE
Feature/sof 6377

### PR DESCRIPTION
**Pull request by azech-hqs**

### CODE REVIEW

File: schema/properties_directory/non-scalar/averaged_potential.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "schemaId": "properties-directory/non-scalar/averaged-potential",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "averaged potential schema",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "macroscopicMinima": {
```
Comment (timurbazhirov) [inline schema/properties_directory/non-scalar/averaged_potential.json]: Should we have the value at the cell edge here too?

File: schema/core/primitive/array_of_numbers.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "schemaId": "core/primitive/array-of-numbers",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "array of numbers",
+    "type": "array",
+    "items": {
+        "type": "number"
+    }
+}
```
Comment (timurbazhirov) [inline schema/core/primitive/array_of_numbers.json:9]: Nit: we have array_of_3_numbers, and could be inheriting from this...

File: manifest/properties.yaml [hunk 3]
```diff
@@ -1,68 +1,68 @@
 convergence_ionic:
   defaults:
     units: eV
-  schemaId: properties-directory-workflow-convergence-ionic
+  schemaId: properties-directory/workflow/convergence/ionic
```
Comment (timurbazhirov) [inline manifest/properties.yaml:4]: Wow - how did this even work?

File: schema/properties_directory/non-scalar/averaged_potential.json (hunk 1 above)
Comment (azech-hqs) [reply schema/properties_directory/non-scalar/averaged_potential.json]: ok yeah, good idea. Is it always the first and last value of the average.x output?

File: manifest/properties.yaml (hunk 3 above)
Comment (azech-hqs) [reply manifest/properties.yaml:4]: it didn't 😄 all `express` unit tests were broken when I updated the latest version of esse. But yeah, we should probably write a test for that, or perhaps there is another way that doesn't require mapping the property slug to the `schemaId`.

File: schema/core/primitive/array_of_numbers.json (hunk 2 above)
Comment (azech-hqs) [reply schema/core/primitive/array_of_numbers.json:9]: Ok, I'll update `array_of_3_numbers` accordingly

File: schema/properties_directory/non-scalar/averaged_potential.json (hunk 1 above)
Comment (timurbazhirov) [reply schema/properties_directory/non-scalar/averaged_potential.json]: Perhaps it is. Maybe we should calculate an average over some small region around the edges to be safe?

**Review by timurbazhirov — APPROVED** [review]

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**